package.json

```diff
@@ -22,12 +22,15 @@
     "build": "rm -rf dist/* tsc-out/* && tsc && rollup -c rollup.config.js",
     "prepare": "npm run build",
     "prepublishOnly": "npm run build",
-    "update": "npx npm-check-updates -u -x @pipobscure/demitasse",
+    "update": "npx npm-check-updates -u -x @pipobscure/demitasse && npm install",
     "playground": "TEMPORAL_PLAYGROUND=1 npm run build && node --experimental-modules --no-warnings --icu-data-dir node_modules/full-icu -r ./dist/playground.cjs",
-    "lint": "eslint . --ext ts,js,mjs,.d.ts --max-warnings 0 --cache \"$@\"",
+    "lint": "eslint . --ext ts,js,mjs,.d.ts --max-warnings 0 --cache \"$@\" && npm run prettier",
     "postlint": "npm run tscheck",
     "prune": "ts-prune -e -i test/tc39 -i \"(lib/index|lib/init|index.d).ts\"",
-    "pretty": "eslint . --ext ts,js,mjs,.d.ts --fix",
+    "prettier": "prettier lib/* ./test/*.mjs ./*.md ./*.json ./*.d.ts ./*.js ./.*.yml --check",
+    "prettierfix": "prettier lib/* ./test/*.mjs ./*.md ./*.json ./*.d.ts ./*.js ./.*.yml  --check --write",
+    "eslintfix": "eslint . --ext ts,js,mjs,.d.ts --fix",
+    "fix": "npm run eslintfix && npm run prettierfix",
```

Comment:
🙏 thank you!

I've been jumping back and forth between this repo and others lately, and having to remember that it's `npm run pretty` was a bit confusing.